### DISCUSSION TITLE
Salah Menampilkan Pesan WhatsApp Notifier dan Symon

### DISCUSSION THREAD

**Discussion by januridp** [hyperjumptech/monika#621] [Symon]
Halo, saya baru aja coba-coba symon, pesan yang tampil di WhatsApp Notifier adalah field query bukan field message di configuration probe dan juga di notif terbalik antara Time dan Target URL

https://imgur.com/a/ri1K2Uo

**Comment by nicnocquee** [comment]
Halo @januridp thank you buat bug reportnya, kita coba fix dulu.
